models/Notification/Service/NotificationService.php

```diff
@@ -28,9 +28,7 @@
 use UnexpectedValueException;
 
 /**
- * @internal
- * With the end of Classic-UI this service will be deprecated.
```

Comment:
So maybe you should `@deprecate` it instead of `@internal`?

Reply:
Thank you for your input.

It feels a little too early for a deprecation, looks like the interface in the studio would be moved here and both should be extending that instead.. I don't know exactly how and when, but for the moment, removing the internal seems right due to the documentation.